src/clojure/nrepl/util/completion.clj

```diff
@@ -3,317 +3,644 @@
 
   The functionality here is experimental and
   the API is subject to changes."
-  {:author "Bozhidar Batsov"
+  {:author "Bozhidar Batsov, Oleksandr Yakushev"
```

Comment:
You might also mention in the docstring that this based on `compliment.lite`, so it's even more visible.